src/protocols/aeternity/libs/AeSdkSuperhero.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
is it needed?

Reply:
Unfortunately yes :/ The changes done in the `getAccounts` method causes the error as there is no `this` reference now.

Reply:
Ok, resolving then